pkg/ema.py

```diff
@@ -47,6 +46,7 @@ def call_ema(id, suid='', message='', alarm='true', test=False):
     phone_url = 'http://191.168.0.106:2226' if not test else 'http://127.0.0.1:5000'
```

Comment:
will these need to be changed for each new phone?

Reply:
i will assume they must ensure the router will always assign this `191.168.0.106` IP to the phone

Reply:
thanks! do we also assume the androidid on line 48 remains the same too?

Reply:
yes~